src/main/java/gift/controller/WishController.java

```diff
@@ -23,34 +23,23 @@ public WishController(WishService wishService) {
 
     @GetMapping
     public ResponseEntity<List<Wish>> getWishes(@LoginMember Member member) {
-        if(member == null){
-            throw new MemberAuthorizationException("Authorization failed");
-        }
+
         return new ResponseEntity<>(wishService.getWishesByMember(member), HttpStatus.OK);
     }
 
     @PostMapping
-    public ResponseEntity<Wish> createWish(@RequestBody WishRequest wishRequest, @LoginMember Member member) {
-        if (member == null) {
-            throw new MemberAuthorizationException("Authorization failed");
-        }
-        Wish wish =wishService.addWish(wishRequest, member);
+    public ResponseEntity<Wish> createWish(@RequestBody ProductIdRequest productIdRequest, @LoginMember Member member) {
+        WishRequest wishRequest = new WishRequest(member.getId(), productIdRequest.getProductId());
+        Wish wish =wishService.addWish(wishRequest);
```

Comment:
전체적인 style 체크도 한번씩 부탁드려요~
```suggestion
        Wish wish = wishService.addWish(wishRequest);
```